src/main/webapp/resources/js/components/project-members/ProjectMembersTable.jsx

```diff
@@ -45,11 +46,11 @@ export function ProjectMembersTable({ projectId }) {
     updateTable();
   }
 
-  const updateProjectRole = ({ id, medataRole }) => (projectRole) =>
-    updateUserRoleOnProject({ projectId, id, medataRole, projectRole });
+  const updateProjectRole = ({ id }) => (projectRole) =>
+    updateUserRoleOnProject({ projectId, id, projectRole });
```

Comment:
Here is an interesting situation.  An administrator can set a project managers metadata role lower than other members - which should not be the case.  Maybe we need to chat with Eric about how to handle that?  Although, I think then the manager can set their own role back to full.  This seems very wrong.

Reply:
Yup we definitely need to discuss this as that all seems very wrong